package.json

```diff
@@ -19,18 +24,24 @@
     ]
   },
   "dependencies": {
+    "assemble": "^0.3.83",
     "auth0-js": "^9.5.1",
+    "components": "^0.1.0",
+    "grunt": "^0.4.5",
     "history": "^4.7.2",
     "react": "^16.3.2",
     "react-dom": "^16.3.2",
-    "react-router-dom": "^4.2.2"
+    "react-router-dom": "^4.2.2",
+    "styles": "^0.2.1"
   },
   "devDependencies": {
     "babel-preset-react-app": "^3.1.1",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
+    "enzyme-to-json": "^3.3.4",
     "eslint": "^4.19.1",
     "jest": "^22.1.4",
+    "node-sass": "^4.9.0",
```

Comment:
O que você acha do PostCSS?
(Sério, só uma idéia haha)

Reply:
To ligado. Mas o Parcel instala essas dependencias sozinho. Sinceramente não to preocupado com isso no momento não.
Acho que rola implementarmos todos os componentes e depois otimizarmos o bundle, tanto de CSS quanto JS.

Reply:
Não sabia que era o parcel quem fazia isso.
Beleza, concordo totalmente. :smiley: